pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go

```diff
@@ -27,37 +26,55 @@ func AddLoadBalancerIPToConnectionDetails(ctx context.Context, iof *runtime.Runt
 		return runtime.NewFatalErr(ctx, "Cannot get composite from function io", err)
 	}
 
-	if comp.Spec.Parameters.Network.ServiceType != "LoadBalancer" {
-		return runtime.NewNormal()
+	k8sservice := &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-%s", comp.GetName(), serviceName),
```

Comment:
I would leave out the `comp.GetName()` here.

If we also want to use this service for port-forwarding it's much simpler for a user, if the service has the same name across the instances. It's also easier for us to document how they can port forward to that service.